frontend/js/src/settings/missing-data/MissingMBData.tsx

```diff
@@ -172,24 +172,6 @@ export default function MissingMBDataPage() {
     }
   };
 
-  const handleClickPrevious = () => {
-    if (currPage && currPage > 1) {
-      setLoading(true);
-      const updatedPage = currPage - 1;
-      setSearchParams({ page: updatedPage.toString() });
-      afterDisplay();
```

Comment:
You can now remove this function `afterDisplay` since it's not being used now.

Reply:
Thanks, totally didn't see those :)